crystal/shivneri/.gitignore

```diff
@@ -0,0 +1,5 @@
+/docs/
```

Comment:
can we please keep this ? as it will cause some unnecessary files/folder to get commited.

Reply:
I prefer to have `.gitignore` files by at the most global level.
I mean to have some `.gitignore` files globally, or in each language folder (it could the avoid duplicated files)

Reply:
understandable, i have removed.